src/components/Parallax.tsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from 'react'
+import ChildrenProp from 'models/ChildrenProp'
+
+export default function ({
+  maxYPixelsDisplacedFromOrigin,
+  children,
+}: ChildrenProp & { maxYPixelsDisplacedFromOrigin: number }) {
+  const [offsetY, setOffsetY] = useState(0)
+  const handleScroll = () => setOffsetY(window.pageYOffset)
+
+  useEffect(() => {
+    window.addEventListener('scroll', handleScroll)
+    return () => window.removeEventListener('scroll', handleScroll)
+  }, [])
+
+  return (
+    <div
+      style={{
+        transform: `translateY(-${
+          (offsetY * maxYPixelsDisplacedFromOrigin) / 8000
```

Comment:
8000 is a magic number. Ideally, there should be no such number. Why is it here after all? Is `offsetY` in pixels? Convert it to rem then.

Reply:
Magic number gone - went with the react-scroll-parallax library instead :)